mautic/mautic
# Pending query optimization

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [y] 
| Related user documentation PR URL      | /
| Related developer documentation PR URL | /
| Issue(s) addressed                     | /



[//]: # ( Required: )
#### Description:
This PR optimizes ineffective DB query fetching number of pending emails.

#### Steps to test this PR:
1. Create a new segment with some contacts.
2. Create a new segment email and choose the segment in the `Contact segment` field.
3. On the email list page you should **see the number** of `pending` **matching the number of contacts** within the segment created in the first step.


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11260"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
